### PR TITLE
Issue #11446: Update AbstractCheckTest to use verifyWithInlineConfigParser

### PR DESCRIPTION
**Pull request by suniti0804**
Issue #11446

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/api/AbstractCheckTest.java [hunk 1]
```diff
@@ -178,7 +178,7 @@ public int[] getRequiredTokens() {
 
         assertWithMessage("Invalid line content")
                 .that(check.getLine(3))
-                .isEqualTo(" * I'm a javadoc");
+                .isEqualTo("*/");
```
Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/api/AbstractCheckTest.java:181]: do `.that(check.getLine(6))` instead

**Review by romani — CHANGES_REQUESTED** [review]
remove from suppressions in pom.xml is you fixed all, if not please update PR description to mention that it is partial fix.

File: src/test/java/com/puppycrawl/tools/checkstyle/api/AbstractCheckTest.java (hunk 1 above)
Comment (suniti0804) [reply src/test/java/com/puppycrawl/tools/checkstyle/api/AbstractCheckTest.java]: done

File: src/test/java/com/puppycrawl/tools/checkstyle/api/AbstractCheckTest.java [hunk 2]
```diff
@@ -177,8 +177,8 @@ public int[] getRequiredTokens() {
             Charset.defaultCharset().name())));
 
         assertWithMessage("Invalid line content")
-                .that(check.getLine(3))
-                .isEqualTo(" * I'm a javadoc");
+                .that(check.getLine(6))
+                .isEqualTo("// violation above");
```
Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/api/AbstractCheckTest.java:181]: sorry i ment to keep `* I'm a javadoc` and just change line number, make line number to be 10. to lee validation same as before.

**Review by romani — CHANGES_REQUESTED** [review]
last

File: src/test/java/com/puppycrawl/tools/checkstyle/api/AbstractCheckTest.java (hunk 2 above)
Comment (suniti0804) [reply src/test/java/com/puppycrawl/tools/checkstyle/api/AbstractCheckTest.java]: done

**Review by romani — APPROVED** [review]
Ok to merge

**Pull request merged**